package.json

```diff
@@ -11,6 +11,7 @@
     "build": "npm run clean && node_modules/webpack/bin/webpack.js",
     "codecov": "cat ./coverage/coverage-final.json | ./node_modules/codecov.io/bin/codecov.io.js",
     "clean": "rm -rf ./static/assets/*",
+    "install-git": "npm install git+https://git@github.com/18F/cg-style.git#master && cd node_modules/cloudgov-style && npm install && npm run build; cd ../../",
```

Comment:
I'm fine calling it whatever. And we should add to docs. Maybe it's a command we should keep around long-term for future times we need it like this?